src/react/deckgl.js

```diff
@@ -58,27 +53,27 @@ const defaultProps = {
   gl: null,
   effects: [],
   onWebGLInitialized: noop,
-  onLayerClick: noop,
-  onLayerHover: noop,
   onAfterRender: noop,
-  onLayerDragStart: noop,
-  onLayerDragMove: noop,
-  onLayerDragEnd: noop,
-  onLayerDragCancel: noop
+  onLayerClick: null,
+  onLayerHover: null
 };
 
 export default class DeckGL extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
     this.needsRedraw = true;
-    this.eventManager = null;
     this.layerManager = null;
     this.effectManager = null;
     autobind(this);
   }
 
   componentWillReceiveProps(nextProps) {
+    this.layerManager.setEventParams({
```

Comment:
Move this duplicate code into `updateLayers` or a new separate function?

`updateLayers` looks like it is already ripe for being renamed `updateLayerManager`